coq/PerformanceDemos/Makefile

```diff
@@ -16,7 +16,7 @@ all:: invoke-coqmakefile
 
 include ../../Makefile.show
 
-WARNINGS:=-notation-overridden
+WARNINGS:=-notation-overridden,-unsupported-attributes
```

Comment:
You need to `make update-_CoqProject` to get this change propagated to the checked-in _CoqProject